cg/services/order_validation_service/rules/case_sample/utils.py

```diff
@@ -33,6 +33,8 @@
 from cg.store.models import Application
 from cg.store.store import Store
 
+import pdb
```

Comment:
What is this import for?

Reply:
It was for testing purpose I forgot to remove it